### ISSUE TITLE
Make `TimeSync` a builtin feature

### ISSUE THREAD

**Issue by zanete** [Green-Software-Foundation/if#823]
**Why**: Sub of #763 
**What**: Move `TimeSync` so it is a builtin feature like `aggregate` rather than a plug in in `builtins`

Blocked by:
- #809 
- #822 

**Scope of work:**
- [ ] TBC

**Comment by jmcook1186** [comment]
Closing as duplicate of #822
